src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccess.tsx

```diff
@@ -0,0 +1,298 @@
+  showButton?: boolean
+  onHide?: Function
+  requestDataStepCallback?: Function
+}
+
+// const LoginPage: React.FunctionComponent<LoginPageProps> =
+const RequestDataAccess: React.FC<RequestDataAccessProps> = (props) => {
+  const { user, token, wikiPage, accessRequirementStatus, accessRequirement, showButton = true, onHide, requestDataStepCallback } = props
+  const [isHide, setIsHide] = useState<boolean>(true)
+  const propsIsApproved = accessRequirementStatus?.isApproved
+  const [isApproved, setIsApproved] = useState<boolean | undefined>(
+    propsIsApproved,
+  )
+  const [submissionState, setSubmissionState] = useState<SUBMISSION_STATE>()
+  const [alert, setAlert] = useState<AlertProps | undefined>()
+  const [isSubmissionCanceled, setIsSubmissionCanceled] = useState<boolean>(false)
+
+  useEffect(() => {
+
+    setIsApproved(propsIsApproved)
+    if (accessRequirementStatus?.currentSubmissionStatus) {
+      setSubmissionState(accessRequirementStatus.currentSubmissionStatus.state)
+      showSubmissionStatusAlert(accessRequirementStatus.currentSubmissionStatus.state)
+    }
+
+  }, [propsIsApproved])
+
+  const gotoSynapseAccessRequirementPage = () => {
+    requestDataStepCallback?.({
+      managedACTAccessRequirement: accessRequirement,
+      step: 1
+    })
+  }
+
+  const onAcceptClicked = async () => {
+    if (
+      accessRequirement.concreteType ===
+        SUPPORTED_ACCESS_REQUIREMENTS.ManagedACTAccessRequirement ||
+      accessRequirement.concreteType ===
+        SUPPORTED_ACCESS_REQUIREMENTS.ACTAccessRequirement
+    ) {
+      if (token) {
+        // !isSubmissionCanceled: if the submission has already been canceled, don't cancel again
+        if (submissionState === SUBMISSION_STATE.SUBMITTED && !isSubmissionCanceled) {
+          const errAlert = {
+            key: 'danger',
+            message: (<>
+              <strong>Error canceling your data access request.</strong><br />
+              Please try again later.
+            </>)
+          }
+          try {
+            const resp:ACTSubmissionStatus | any = await cancelDataAccessRequest(accessRequirementStatus?.currentSubmissionStatus!.submissionId, token)
+            if (resp.state === SUBMISSION_STATE.CANCELLED) {  // successfully cancelled
+              setAlert({
+                key: 'success',
+                message: (<strong>Your data access request has been canceled.</strong>)
+              })
+              setIsSubmissionCanceled(true)
+            } else {
+              setAlert(errAlert)
+            }
+          } catch (e) {
+            console.log("RequestDataAccess: error canceling data access request:", e)
+            setAlert(errAlert)
+          }
+        } else {
+          gotoSynapseAccessRequirementPage()
+        }
+      } else {
+        requestDataStepCallback?.({
+          managedACTAccessRequirement: accessRequirement,
+          step: 4
+        })
+      }
+
+      // PORTALS-1483: and close the dialog.
```

Comment:
Just delete this code.  You're right that requesting access should not close the dialog (PORTALS-1483 is applicable when it sends the user off to request access in another window).

Reply:
Done.